validator/validator/metrics.py

```diff
@@ -42,3 +43,15 @@ def calculate_score(self, uid: int) -> float:
             0.0,
             self.baseline_averages[uid] - self.model_averages[uid]
         ) * self.similarity_averages[uid]
+
+    def __getstate__(self):
+        return {
+            "baseline_averages": self.baseline_averages,
```

Comment:
baseline_average should be a part of the Contest, rather than being set per UID. A placeholder value for it works for now until we get a calculation for the actual value.